src/res_printer.ml

```diff
@@ -99,6 +99,15 @@ let hasCommentBelow tbl loc =
   | [] -> false
   | exception Not_found -> false
 
+let rec hasNestedJsx expr =
+  match expr.Parsetree.pexp_desc with
+  | Pexp_construct ({txt = Longident.Lident "[]"}, _) -> false
+  | Pexp_construct
+      ( {txt = Longident.Lident "::"},
+        Some {pexp_desc = Pexp_tuple (hd :: [tail])} ) ->
```

Comment:
nit: this is normally written `[hd, tail]`.

Reply:
Hmmm, this gives me:

```
File "src/res_printer.ml", line 106, characters 39-44:
106 |         Some {pexp_desc = Pexp_tuple ([hd, _])} ) ->
                                             ^^^^^
Error: This pattern matches values of type 'a * 'b
       but a pattern was expected which matches values of type
         Parsetree.expression
```

My OCaml knowledge is not enough to understand why is it different. Perhaps, type inference fails somehow it the `[hd, tail]` case. Anyway, I’ve stolen this pattern from `ParsetreeViewer.collectListExpressions`.

Reply:
Sorry `[hd; tail]`

Reply:
> > As a final step, before and after: make bench ... If you are on macos, that is
> 
> I’m on Linux and have no access to a macOS machine, unfortunately. Would someone help with running `bench`?

Will do once the PR is final.

Reply:
> Sorry `[hd; tail]` not `[hd, tail]`

Muscle memory :) Fixed it.